src/org/jitsi/service/neomedia/MediaStream.java

```diff
@@ -254,6 +280,28 @@ public void addDynamicRTPPayloadTypeOverride(byte originalPt,
     public List<Long> getRemoteSourceIDs();
 
     /**
+     * Adds the remote {@code MediaStreamTrack}s to this {@code MediaStream}.
+     */
+    public void addRemoteTrack(MediaStreamTrack mediaStreamTrack);
+
+    /**
+     * Gets the remote {@code MediaStreamTrack} that matches the given SSRC.
+     *
+     * @param ssrc the SSRC to match the {@code MediaStreamTrack} against. It
+     * can be the primary SSRC or the RTX SSRC.
+     *
+     * @return the remote {@code MediaStreamTrack} that matches the given SSRC,
+     * or null if there's no matching {@code MediaStreamTrack}.
+     */
+    public MediaStreamTrack getRemoteTrack(long ssrc);
+
+    /**
+     * Clears the remote {@code MediaStreamTrack}s associated with this
+     * {@code MediaStream}.
+     */
+    public void clearRemoteTracks();
```

Comment:
Thank you @lyubomir, you're right. I have updated the code accordingly. Please let me know if this is better or if it needs further modifications.